api/v1alpha1/perconaservermysql_types.go

```diff
@@ -193,6 +193,7 @@ func (s *PodSpec) GetInitImage() string {
 type PMMSpec struct {
 	Enabled                  bool                        `json:"enabled,omitempty"`
 	Image                    string                      `json:"image"`
+	MysqlParams              string                      `json:"mysqlParams,omitempty"`
```

Comment:
please rename to `MySQLParams`

Reply:
Yep, it's no problem to rename. Do we have any negotiation about it? In pxc we have for proxysql in pmm
`ProxysqlParams           string                      `json:"proxysqlParams,omitempty"``

but for 
`ProxySQL           AppStatus          `json:"proxysql,omitempty"``

Reply:
fixed